BRFCore/Code/UI/UIColor+BR.m

```diff
@@ -0,0 +1,67 @@
+//
+//  UIColor+BR.m
+//  BlueRocketFuel
+//
+//  Created by Estevan Hernandez on 11/2/16.
+//  Copyright © 2016 Blue Rocket. All rights reserved.
+//
+
+#import "UIColor+BR.h"
+
+@implementation UIColor (BR)
++ (UIColor *)colorFromRGBHex:(uint64_t)rgb {
```

Comment:
That is a good point..
You're also right about BRStyle, I didn't see the methods in there.
I'll close the pull request.